pandas_gbq/tests/test_gbq.py

```diff
@@ -19,9 +19,9 @@
 import pandas.util.testing as tm
 from pandas.compat.numpy import np_datetime64_compat
 
-PROJECT_ID = None
-PRIVATE_KEY_JSON_PATH = None
-PRIVATE_KEY_JSON_CONTENTS = None
+PROJECT_ID = os.getenv('PROJECT_ID')
```

Comment:
I think the Google library pulls from `GOOGLE_CLOUD_PROJECT` (more confident about the newer library, which this package doesn't use yet). So you could either replicate that variable, or pass `None` through and the Google library should pick it up